addons/youtube/index.js

```diff
@@ -28,6 +28,8 @@ export default class YouTube extends React.Component {
   render() {
     const { baseUrl, content } = this.props
 
+    console.log(this.props.name)
```

Comment:
✂️

Reply:
for sure! left this here just for demonstration purposes.